src/blog/announcing-papyrs-blog-on-web3.md

```diff
@@ -0,0 +1,156 @@
+tags: "#startup #programming #web3 #blockchain"
+image: "https://miro.medium.com/max/1400/1*0aVC6yVZLehL990fUJDazw.png"
+canonical: "https://medium.com/dfinity/dynamically-create-canister-smart-contracts-in-motoko-d3b38a748c07"
+---
+![https://6zvwc-sqaaa-aaaal-aalma-cai.raw.ic0.app/images/social-image.jpg?token=CLmDdhvrv_oiB27vvPeb9](https://6zvwc-sqaaa-aaaal-aalma-cai.raw.ic0.app/images/social-image.jpg?token=CLmDdhvrv_oiB27vvPeb9)
+
+
+So, I have developed a new open-source, privacy-first, decentralized blogging platform that lives 100% on chain and today I am making it available to anyone. I called it: [Papyrs](https://app.papy.rs).
+
+* * *
+
+## What a ride!
+
+Last year, I was lucky enough to receive grants from the DFINITY foundation to port our web editor for slides, [DeckDeckGo](https://deckdeckgo.com/), to the [Internet Computer](https://smartcontracts.org/).
+
+While I completed all milestones, I did not migrated it yet.
+
+Along the way, I had the idea of extending the platform to support documents. One thing leading to another, I ended developing my own rich text editor library, [Stylo](https://stylojs.com/), and decided to focus first on this new format hence me launching today a new blogging platform.
+
+On the one hand, thanks to this approach, I limit the number of functions I launch on the IC, which allows me to gain experience to later deploy our presentation editor more serenely.
+
+On the other hand, I like to develop new ideas and thus double the fun 😜.
+
+* * *
+
+## Manifesto
+
+So, why launching another blogging application? After-all there are already many platforms that allow you to write articles isn't 🤷‍♂️?
+
+Sure there is but none are following the guidelines that drive me.
+
+* * *
+
+### Decentralization & blockchain
+
+The concept of decentralization - or web3 - is linked to a certain form of romanticism for developers that [Chris Dixon](https://twitter.com/cdixon) summarizes in following terms:
+
+*The lesson is that when you compare centralized and decentralized systems you need to consider them dynamically, as processes, instead of statically, as rigid products. Centralized systems often start out fully baked, but only get better at the rate at which employees at the sponsoring company improve them. Decentralized systems start out half-baked but, under the right conditions, grow exponentially as they attract new contributors ([source](https://twg.io/whiteboard/why-decentralization-matters/)).*
+
+As a programmer my self, this idea is appealing. I also think that blockchain technology might lead to interesting new perspective for the future.
+
+I can notably imagine that consensus and other certification mechanism could potentially be useful to proof the validity of the articles that would be shared with Papyrs - i.e. could be used to prevent fake news and misinformation.
```

Comment:
```suggestion
I can for instance imagine that consensus and other certification mechanisms could potentially be useful to prove the validity of the articles shared with Papyrs - i.e. could be used to prevent fake news and misinformation.
```